docs/usage.md

```diff
@@ -29,6 +29,18 @@ system:
 sudo curl -L --output /usr/local/bin/pie https://github.com/php/pie/releases/latest/download/pie.phar && sudo chmod +x /usr/local/bin/pie
 ```
 
+### Docker installation
+
+PIE is published as binary-only Docker image, so you can install it easily during your Docker build:
```

Comment:
We should probably explicitly note that this image doesn't have any dependencies; it's implied by "binary-only", but I think it would be clearer for people reading the docs.

Reply:
Thanks, looks good